CHANGELOG.md

```diff
@@ -0,0 +1,24 @@
+## 1.0.6
+- Marked getErrorType in Error as deprecated since 2.0.0
```

Comment:
This indicates that the method is deprecated since next major version and not within this version. 
And also that it is planned for removal in 3.0.0.
Is that intended here?

Reply:
Bump, also 👍 I pointed that in other PR too

Reply:
My intension is that the next minor release version contains information about deprecation since the next major version.

Reply:
@emilleszczak2 usually planning future deprecations on minor versions is kinda weird, you already decided to deprecate it now when planning the release of 1.0.6, right?
Your originated intention was to remove the method in 1.0.6 but went with a suggestion to deprecate it and that's fine.
IMO you don't need to postpone it till 2.0.0 release just to announce.
---
Don't get me wrong. I'm just trying to gather it all into sensible releases.

Reply:
@brzuchal I'm fine with this :)

I decided to mark it as deprecated in 1.0.6 and remove it in 2.0.0. Are you OK with that?

Reply:
IMO that's how it should be done!

Reply:
Great. Let's do it.